requirements/COMP_MANA_AURORA-0004/startup-priority-demo/test-expected.log

```diff
@@ -0,0 +1,10 @@
+[Function_1] Startup
```

Comment:
Startup, at least the initial one, is automatic. But COMP_MANA_0004 also mentions the shutdown. We have the STOP interfaces. It would be nice to either modify this model, or to create another one, which has a component, e.g., Execution Service (name taken from the description in D5.1), which issues STOP commands in a sequence. In case of doubt, let's talk.

Reply:
Added new test for shutdown order